nixos/tests/installer.nix

```diff
@@ -291,7 +291,7 @@ let
           # builds stuff in the VM, needs more juice
           virtualisation.diskSize = 8 * 1024;
           virtualisation.cores = 8;
-          virtualisation.memorySize = 1536;
+          virtualisation.memorySize = 2047;
```

Comment:
Is there a reason this is just one MB shy of 2048?

Reply:
https://github.com/NixOS/nixpkgs/commit/aa3b13549fbd4d5da27bdf07b6cb66555e837b8e
https://hydra.nixos.org/build/24983009
on i686-linux

`machine# qemu: at most 2047 MB RAM can be simulated`